iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -33,6 +38,14 @@ class MessageCell: UICollectionViewCell {
 
     private func initializeViews() {
         self.contentView.addSubview(self.content)
+
+        ConversationsManager.shared.$activeConversation
+            .removeDuplicates()
+            .mainSink { [unowned self] activeConversation in
+                // If this cell's conversation becomes active,
+                // then start message consumption if needed.
+                self.handleDetailsShown(self.detailVC.view.alpha == 1)
+            }.store(in: &self.subscriptions)
```

Comment:
This fixes the issue where a message wouldn't start being consumed when you slid over to a new conversation.